x-pack/plugins/actions/server/create_execute_function.ts

```diff
@@ -41,7 +41,7 @@ export function createExecutionEnqueuerFunction({
   ) {
     if (isESOUsingEphemeralEncryptionKey === true) {
       throw new Error(
-        `Unable to execute action due to the Encrypted Saved Objects plugin using an ephemeral encryption key. Please set xpack.encryptedSavedObjects.encryptionKey in kibana.yml`
+        `Unable to execute action due to the Encrypted Saved Objects plugin using an ephemeral encryption key. Please set xpack.encryptedSavedObjects.encryptionKey in kibana.yml manually or by using bin/kibana-encryption-key`
```

Comment:
```suggestion
        `Unable to execute action because the Encrypted Saved Objects plugin uses an ephemeral encryption key. Please set xpack.encryptedSavedObjects.encryptionKey in the kibana.yml or  use the bin/kibana-encryption-key command.`
```

Reply:
Thanks @gchaps.  Pushed with 8d7dad6